pkg/util/xorm/go.mod

```diff
@@ -1,13 +1,20 @@
 module xorm.io/xorm
```

Comment:
should we rename this? 
Something like `github.com/grafana/grafana/pkg/util/xorm`

Reply:
good question. i'll try that and see if it still works. my guess is that it might break imports

Reply:
Oh yeah that's true. But if we will publish this module how will it be like for a consumer? I am asking this to have an idea for our future packages.

Reply:
it seems like `replace` works in `go.sum`. see: [14735ec](https://github.com/grafana/grafana/pull/83191/commits/14735ec19f6a06abc40be8603bbbe7f5081b30d4)